docs/use_cases/model_applications/s2s_mjo/UserScript_obsCFSR_obsOnly_MJO_ENSO.py

```diff
@@ -81,24 +81,43 @@
 
 ##############################################################################
 # MET Configuration
-# ---------------------
+# -----------------
+#
+# METplus sets environment variables based on user settings in the METplus
+# configuration file. See :ref:`How METplus controls MET config file settings<metplus-control-met>` for more details.
 #
-# METplus sets environment variables based on the values in the METplus configuration file.
-# These variables are referenced in the MET configuration file. **YOU SHOULD NOT SET ANY OF THESE ENVIRONMENT VARIABLES YOURSELF! THEY WILL BE OVERWRITTEN BY METPLUS WHEN IT CALLS THE MET TOOLS!** If there is a setting in the MET configuration file that is not controlled by an environment variable, you can add additional environment variables to be set only within the METplus environment using the [user_env_vars] section of the METplus configuration files. See the 'User Defined Config' section on the 'System Configuration' page of the METplus User's Guide for more information.
+# **YOU SHOULD NOT SET ANY OF THESE ENVIRONMENT VARIABLES YOURSELF! THEY WILL BE OVERWRITTEN BY METPLUS WHEN IT CALLS THE MET TOOLS!**
 #
+# If there is a setting in the MET configuration file that is currently
+# not supported by METplus you’d like to control, please refer to:
+# :ref:`Overriding Unsupported MET config file settings<met-config-overrides>`
 #
+# .. dropdown:: mjo_enso_driver
+#
+#   .. literalinclude:: ../../../../parm/use_cases/model_applications/s2s_mjo/UserScript_obsCFSR_obsOnly_MJO_ENSO/mjo_enso_driver.py
 
 ##############################################################################
 # Python Scripts
```

Comment:
@lisagoodrich The title of this section is currently "Python Scripts".  The title of this section should be "Python Embedding".  However, I think the information contained in this section may be intended for the new "User Scripting" section.  I am unsure though.  Please check with the METplus Team Member/Owner (@georgemccabe is listed in the spreadsheet as the contact) and update this file as necessary.  Thanks!

Reply:
@georgemccabe Could you please take a look at this and tell me how best to proceed with this section?  Thanks!

Reply:
@jprestop, @lisagoodrich, yes, the information about the `mjo_enso_driver.py` script should go under the "User Scripting" section. There is no "Python Embedding" information relevant for this use case.

I also notice that the "MET Configuration" includes information about the `mjo_enso_driver.py` script, but it should not. It looks like there are no MET Configuration files used in this case.

@CPKalb could confirm or deny that what I'm saying is correct here.

Reply:
@georgemccabe, @jprestop, @lisagoodrich yes, this is correct.  

As a heads up, the MJO ENSO case was added to METplus by Cristiana Stan.  George and I helped her get it merged into METplus.  I can answer questions regarding the processing/steps, but I don't know the science of this use case too well.

Reply:
It looks "MET Configuration" section only says "There are no MET tools used in this use case." in this branch.  @georgemccabe were you looking at the file in a different branch?

Reply:
@jprestop, I saw this by clicking on the file path in this discussion, but looking at the feature_2741_dropdowns branch explicitly, I do see that the section reflects what you say. Maybe this was fixed in a subsequent commit from the start of this discussion?

Reply:
@georgemccabe Interesting behavior.  I just did the same thing you did and I see it now too.  I was looking under the "Files Changed" for this PR and I see what I originally described.

Reply:
Okay, I've 
1. Changed the title from Python Scripting to Python Embedding.
2. Under Python Embedding I have:  Python Embedding  is not used in this use case.
3. Moved the MJO ENSO information to User Scripting.
I think this is under control. @jprestop, @georgemccabe @CPKalb holler if I should be reaching out to Cristiana Stan.  (I don't know how to contact her.) I don't think I need to because I kept the MJO ENSO information, I just moved the section it was in.
Thanks for all your help in getting this under control.